tensorflow/compiler/xla/service/hlo_verifier.cc

```diff
@@ -931,6 +931,9 @@ Status CheckMixedPrecisionOperands(const HloInstruction* instruction) {
     case HloOpcode::kSort:
     case HloOpcode::kTuple:
     case HloOpcode::kWhile:
+    case HloOpcode::kBatchNormGrad:
+    case HloOpcode::kBatchNormInference:
+    case HloOpcode::kBatchNormTraining:
```

Comment:
The reason for having these here is different than what the comment says right?  These ops are not "passthrough" like `kWhile`, they genuinely have mixed-precision operands?

Reply:
Hmmm...I think I missed the comments. However, the reason I added these is becoz I did get an error from the `hlo_verifier` without this change. I might have missed something. I'll take a look.

Reply:
You are right..I am not sure what it was..I removed these cases. These are incorrect. Sorry for the oversight.